go-controller/pkg/ovn/egressip_test.go

```diff
@@ -152,6 +151,48 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 		return statuses[0].Node
 	}
 
+	//sets the Networks field of the logicalRouter given by name
+	setupLogicalRouterPortNetworks := func(nbDatabase []libovsdbtest.TestData, routerPortName string, networks []string) {
+		for _, object := range nbDatabase {
+			if lrp, ok := object.(*nbdb.LogicalRouterPort); ok {
+				if lrp.Name == routerPortName {
+					lrp.Networks = networks
+				}
+			}
+		}
+	}
+
+	changeRouterPolicies := func(nbDatabase []libovsdbtest.TestData, routerName string, policies []string) {
+		for _, object := range nbDatabase {
+			if lr, ok := object.(*nbdb.LogicalRouter); ok {
+				if lr.Name == routerName {
+					lr.Policies = policies
+				}
+			}
+		}
+	}
+
+	changeLogicalSwitchPortOptions := func(nbDatabase []libovsdbtest.TestData, lspName string, options map[string]string) {
+		for _, object := range nbDatabase {
+			if lsp, ok := object.(*nbdb.LogicalSwitchPort); ok {
+				if lsp.Name == lspName {
+					lsp.Options = options
+				}
+			}
+
+		}
+	}
+
+	changeLogicalRouterNat := func(nbDatabase []libovsdbtest.TestData, lrName string, nat []string) {
+		for _, object := range nbDatabase {
+			if lr, ok := object.(*nbdb.LogicalRouter); ok {
+				if lr.Name == lrName {
+					lr.Nat = nat
+				}
+			}
+		}
+	}
+
```

Comment:
I honestly don't think readability is improved from any of this. It kind of just makes the tests more difficult to read, could you revert this?

Reply:
done